pyautomagic/src/Subject.py

```diff
@@ -0,0 +1,78 @@
+import ntpath
```

Comment:
for consistency with other code in the repo, use os.path here. It should do the same thing.